pkg/operator/datasources.go

```diff
@@ -230,6 +230,7 @@ func (op *Reporting) handlePrometheusMetricsDataSource(logger log.FieldLogger, d
 	// run the import
 	results, err := importer.ImportFromLastTimestamp(context.Background())
 	if err != nil {
+		op.eventRecorder.Event(dataSource, v1.EventTypeWarning, "FailedPrometheusQuery", "Unable to import metrics after Prometheus query failure. Check the reporting-operator container logs for more information.")
```

Comment:
Also, we could include the actual error in the event's message, but it can pretty long and difficult to parse, especially when we get an error that's in this kind of form:
```
Get https://thanos-querier.openshift-monitoring.svc:8080/api/v1/query?query=up&time=2020-07-29T17%3A50%3A52.641742567Z: dial tcp 172.30.48.65:8080: connect: no route to host
```